utils/session.ts

```diff
@@ -11,3 +11,15 @@ export function useSession() {
     return InlineResponse2004ToJSON(body);
   });
 }
+
+export function formatRoles(roles: string) {
+  const normalizedRoles = roles
+    .replace(/[^,]+\//g, "")
+    .toLowerCase()
+    .split(",");
+  if (normalizedRoles.includes("administrator")) return "管理者";
+  if (normalizedRoles.includes("instructor")) return "教育者";
+  if (normalizedRoles.includes("teaching assistant")) return "TA";
+  if (normalizedRoles.includes("student")) return "学習者";
+  return "不明なロール";
```

Comment:
不明なロールに相当するロールはシステム内で許容すべきでないので不要と思います。

Reply:
```suggestion
  return "";
```

何にも該当しなかった場合もreturnするべきかと思ったのでreturnを置きましたが、「不明なロール」という名称は必要と思っていなかったです

if returnしているからこういう書き方になるので、objectを連想配列のように使って書く方が良いなと思いました